### PR TITLE
HPCC-20887 Prevent old style login from displaying

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [ ] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by GordonSmith** [comment]
@cloLN ready for cherry picking
@miguelvazq @kunalaswani please review

**Comment by GordonSmith** [comment]
Do not pull - I need to test against an environment with no security...

File: esp/src/package.json [hunk 1]
```diff
@@ -16,10 +16,10 @@
     "copy-res-stub_htm": "cpx \"./stub.htm\" ./build/",
     "copy-res": "run-p copy-res-es6-promise copy-res-eclwatch-img copy-res-eclwatch-ecl copy-res-dojo copy-res-dojox copy-res-TopoJSON copy-res-font-awesome copy-res-stub_htm",
     "compile": "tsc",
-    "compile-watch": "run-s compile -- -w",
+    "compile-watch": "npm run compile -- -w",
```
Comment (miguelvazq) [inline esp/src/package.json]: These are probably by mistake but not needed in session fix.

File: esp/src/eclwatch/stub.js [hunk 2]
```diff
@@ -30,12 +30,13 @@ define([
     ESPUtil, Utility, LockDialogWidget,
     entities, Toaster) {
 
-        var IDLE_TIMEOUT = cookie("ESPSessionTimeoutSeconds") * 1000;
+        var espTimeoutSeconds = cookie("ESPSessionTimeoutSeconds") || 600;  // 10 Minutes?
```
Comment (miguelvazq) [inline esp/src/eclwatch/stub.js]: Changing the variable name is fine but no `||` should be allowed. This is a completely configured time and we do not want to limit the user to anything but the configured time. The administrator should make the decision of the configured time not the front end code.

**Review by miguelvazq — CHANGES_REQUESTED** [review]

File: esp/src/eclwatch/stub.js (hunk 2 above)
Comment (GordonSmith) [reply esp/src/eclwatch/stub.js]: When the cached page is loaded, there are no cookies and espTimeoutSeconds === undefined, which means the idle watcher is never getting set, which in turn means it errors when a doIdle topic is published.

Comment (GordonSmith) [reply esp/src/eclwatch/stub.js]: If it is set then the `|| 600` never comes into play.

File: esp/src/package.json (hunk 1 above)
Comment (GordonSmith) [reply esp/src/package.json]: There ok to include.

File: esp/src/lws.config.js [hunk 3]
```diff
@@ -1,10 +1,13 @@
-const debugServerIP = "192.168.99.103";
+const debugServerIP = "10.173.160.101";
```
Comment (miguelvazq) [inline esp/src/lws.config.js]: Do we need to commit this IP? I think the routes are important.

File: esp/src/src/ESPRequest.ts [hunk 4]
```diff
@@ -105,6 +130,21 @@ class RequestHelper {
     }
 
     send(service, action, params?) {
+        // console.log(`service: ${service}\naction: ${action}\nhasServerSetCookie: ${this.hasServerSetCookie()}\nhasAuthentication: ${this.hasAuthentication()}\nisAuthenticated: ${this.isAuthenticated()}`);
```
Comment (miguelvazq) [inline esp/src/src/ESPRequest.ts]: remove console log?

**Review by miguelvazq — APPROVED** [review]

**Pull request merged**